src/js/modules/scrollTopArrow.js

```diff
@@ -0,0 +1,27 @@
+import { throttle } from "throttle-debounce";
+
+const SCROLL_DELAY = 250;
+const PAGE_TOP = 0;
+let button = document.getElementById("buttonScrollTop");
+let viewportHeight = document.documentElement.clientHeight;
+
+button.addEventListener("click", ()=> window.scrollTo({
+	top: PAGE_TOP,
+	behavior: "smooth"
+}));
+
+const throttledMoveToTop = throttle(SCROLL_DELAY, moveToTop);
+
+function moveToTop() {
+	const scrollHeight = window.pageYOffset;
+
+	if(scrollHeight > viewportHeight) {
+		button.style.opacity = "0.7";
```

Comment:
use class in css for it (visibility + opacity) instead of JS

Reply:
I don't need visibility here. Button disabled and opacity: 0. Focus doesn't work.
Done